packages/http/src/mocker/generator/JSONSchema.ts

```diff
@@ -1,10 +1,12 @@
 import * as faker from 'faker/locale/en_US';
-import { cloneDeep } from 'lodash';
 import { JSONSchema } from '../../types';
 
 import * as jsf from 'json-schema-faker';
 import * as sampler from 'openapi-sampler';
-import { Either, tryCatch, toError } from 'fp-ts/Either';
+import { Either, tryCatch, toError, right } from 'fp-ts/Either';
```

Comment:
We usually import everything 
```suggestion
import * as E from 'fp-ts/Either';
```